libs/openFrameworks/graphics/ofImage.cpp

```diff
@@ -182,16 +182,21 @@ static int getJpegOptionFromImageLoadSetting(const ofImageLoadSettings &settings
 template<typename PixelType>
 static bool loadImage(ofPixels_<PixelType> & pix, const of::filesystem::path& _fileName, const ofImageLoadSettings& settings){
 	ofInitFreeImage();
-
+	
+#if defined(TARGET_WIN32) || defined(__MINGW64__) || defined(__CYGWIN__) || defined(TARGET_WINVS)
+	#define OF_TARGET_WINDOWS
 	auto uriStr = _fileName.string();
+#else
+	auto uriStr = _fileName;
+#endif
```

Comment:
this creates a situation where on windows `uriStr` is a `std::string`, whereas elsewhere it is an `of::filesystem::path` — which happens to be OK because both class support the later `.c_str()` class but is confusing with the `Str` suffix. moreover, the destination type for URIetc functions being `const char *` (all future uses of `uriStr` are with a `.c_str()` call) it would be simpler to call 
```
uriStr = _fileName.string().c_str(); // char * everywhere
```
however: when the `_fileName` is a windows path containing inconvertible unicode characters, the `.string()` method will throw an exception. all `.string()` calls should be try/catched.